components/QueueCard.js

```diff
@@ -27,35 +27,36 @@ const QueueCard = ({ queue, courseName, onDelete, onUpdate, ...rest }) => {
     onUpdate()
   }
 
+  const title = courseName || queueName
+  const showQueueNameInBody = !!courseName
+
   return (
     <Card className="queue-card" {...rest}>
       <CardBody>
-        <CardTitle className="d-flex align-items-center">
-          {courseName}
-          <ShowForCourseStaff courseId={queue.courseId}>
-            <Button
-              color="danger"
-              size="sm"
-              className="mr-1 ml-auto"
-              outline
-              onClick={handleDelete}
-            >
-              Delete
-            </Button>
-          </ShowForCourseStaff>
-          <ShowForCourseStaff courseId={queue.courseId}>
-            <Button
-              color="primary"
-              size="sm"
-              className="mr-0 ml-1"
-              outline
-              onClick={handleUpdate}
-            >
-              Edit
-            </Button>
-          </ShowForCourseStaff>
+        <CardTitle className="d-flex flex-wrap align-items-center">
+          <span className="mb-2 mr-auto pr-3">{title}</span>
```

Comment:
Would it be possible to have something like "showQueueNameInBody" (maybe a showCourseNameInBody) here? For a course page, it seems a bit redundant to show the course name at the top of the page, and then at the top of every queue card. If this route is taken, may be nice to have queueName bumped up to be the CardTitle instead of just the CardSubtitle on the course homepage

Reply:
I took care of this in the latest commit actually, I planned for that in the beginning but forgot about it until the PR.

Reply:
Oh yeah didn't connect that title here was from line 30, my bad 😱